.python-version

```diff
@@ -0,0 +1 @@
+3.6.5
```

Comment:
What's the purpose of this file?

Reply:
it's for pyenv

Reply:
But we have the Pipfile that dictate the version of Python